### PR TITLE
OSDOCS#10138: Adding RNs for OSSO 1.3.0

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-10138

Link to docs preview:

https://77330--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/index.html
https://77330--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-configuring.html
https://77330--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html
https://77330--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-uninstalling.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ingvagabund @kasturinarra Can you please review?

I noticed another bug attached to the advisory, so I added it under bug fixes. Let me know if this simple note is sufficient, or if we want to get into more detail about why these errors were showing up in the logs, etc.

Preview: https://77330--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html#secondary-scheduler-operator-release-notes-1.3.0_nodes-secondary-scheduler-release-notes

**Comment by ingvagabund** [comment]
The bug fix can be ignored. It's just a cosmetic one. It does not cause any issues since osso does not use the infra object. It's the library-go code that reads it in one of the auxiliary functions/methods.

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 1]
```diff
@@ -10,7 +10,41 @@ The {secondary-scheduler-operator-full} allows you to deploy a custom secondary
...
+Issued: 2024-07-01
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.3.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3637[RHSA-2024:3637]
+
+[id="secondary-scheduler-operator-1.3.0-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* You can now install and use the {descheduler-operator} in an {product-title} cluster running in FIPS mode.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:27]: Ooo bad copy paste sorry, good catch!

**Comment by bergerhoffer** [comment]
@kasturinarra @ingvagabund Removed the bug and fixed the operator name for the FIPS note, if you can take another look whenever you get a chance.

Preview: https://77330--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html

**Comment by ingvagabund** [comment]
lgtm

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.17` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.16. And any PR going into `main` must also target the latest version branch (enterprise-4.17).

If the update in your PR does NOT apply to version 4.17 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**